### DISCUSSION TITLE
WebRTC connection issues

### DISCUSSION THREAD

**Discussion by dillanspencer** [bluenviron/mediamtx#2467] [Questions]
### Question

Hello everyone,

I'm currently encountering challenges with my WebRTC client setup. Specifically, I'm observing frequent and immediate disconnections shortly after connection attempts. Here's a brief overview of my setup:

Hardware: Raspberry Pi
Networking: Operating behind a Peplink router
Connectivity Solution: I've incorporated a TURN server to facilitate a peer-to-peer connection to my Raspberry Pi device.
Although a connection appears to be established initially, it disconnects almost immediately. Consequently, I never receive a video feed in my player.

This screenshot shows the state of my connection:
![Screenshot 2023-10-05 at 11 14 44 AM](https://github.com/bluenviron/mediamtx/assets/46987282/32ee9404-f180-45b4-b1ca-192656fe0081)

To provide further context:

I'm running four streams simultaneously.
The Raspberry Pi device is currently located on a different continent, which may be a contributing factor to the issue.
If anyone has insights or suggestions on potential solutions or troubleshooting steps, they would be greatly appreciated.

Thank you!

**Comment by aler9** [comment]
Hello, without server logs it's impossible to understand where the problem is. Please reopen the discussion / issue and attach server logs as stated in the instructions.

[a comment by an automated account is not shown]
